tests/functional/gpu_multi_container/docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+services:
```

Comment:
This approach isn't going to work in Harness right?

Reply:
🤔 maybe if for these GPU tests we could get a Harness nodepool that runs on the base OS and not in K8s this would work actually (https://developer.harness.io/docs/continuous-integration/use-ci/set-up-build-infrastructure/vm-build-infrastructure/set-up-an-aws-vm-build-infrastructure -- looks like it just became deprecated 🙃 )

Reply:
> This approach isn't going to work in Harness right?

Thats ok. I need tests now and we don't yet have GPU harness. But it's a problem that I have to solve.

Reply:
On the other side, we can (and should) add acceptance tests which include custom models API + LRS Operator + DRUM.